tests/test_backend.py

```diff
@@ -128,5 +128,21 @@ def test_validate_fail():
     _ = validate_data_format_str('weird_string')
 
 
+@pytest.mark.parametrize('frame_length', [1024, 2048])
+@pytest.mark.parametrize('frame_step', [256, 512])
+@pytest.mark.parametrize('image_size', [256, 512, 1024])
+def test_parallel_stft_should_be_equal_to_tf_stft_on_cpu(frame_length, frame_step, image_size):
```

Comment:
I don't think it's a bad name, but for the consistency with other functions, I'd like `test_parallel_stft_correctness` slightly more.